po/glossary/it.po

```diff
@@ -38,7 +39,7 @@ msgstr ""
 
 #. "Opening and closing quote symbols and optionally their key combos like [altgr]+[Y]/[X]. Define the preferred style of quotation, see https://en.wikipedia.org/wiki/Quotation_mark#Summary_table"
 msgid "\"\""
-msgstr ""
+msgstr "\"\""
```

Comment:
@Padanian No, please see in the main file. @MocioF used the nicer looking «».

Reply:
Makes no sense to me. The translation of two double apexes is two double apexes.

Reply:
No, typographical correct quotation is country specific. Didn't you read the [link in the comment](https://en.wikipedia.org/wiki/Quotation_mark#Summary_table)?
en_US: "…", but
en_GB: '…', and other european languages mostly «…» or „…“ and https://github.com/Gnucash/gnucash/blob/maint/po/it.po uses «…».

Reply:
> No, typographical correct quotation is country specific. Didn't you read the [link in the comment]

No, I didn't. No other quotation marks other than " are available on IT-it or CH-it keyboards. 
Mind you, msgstr "\"\"" is something very well removed from «…».

Reply:
> No other quotation marks other than " are available on IT-it or CH-it keyboards.

Did you try the example from the comment: `[altgr]+[Y], [altgr]+[X]`?